datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemCheckBoxViewHolderFactoryInstrumentedTest.kt

```diff
@@ -136,4 +136,46 @@ class QuestionnaireItemCheckBoxViewHolderFactoryInstrumentedTest {
     assertThat(answer.size).isEqualTo(1)
     assertThat(answer[0].valueBooleanType.value).isTrue()
   }
+
+  @Test
+  @UiThreadTest
+  fun shouldSetAnswerToFalse() {
```

Comment:
```suggestion
  fun click_shouldSetCheckBoxUnchecked() {
```

also rename the previous test case `shouldSetQuestionnaireResponseItemComponentAnswer` to `click_shouldSetAnswerTrue`.

Basically you should have 4 cases:

```
fun click_shouldSetCheckBoxChecked
fun click_shouldSetAnswerTrue
fun click_shouldSetCheckBoxUnchecked
fun click_shouldSetAnswerFalse
```